test/services/role/services/userPermissions.test.js

```diff
@@ -0,0 +1,94 @@
+const { expect } = require('chai');
+const app = require('../../../../src/app');
+const testObjects = require('../../helpers/testObjects')(app);
+
+const userPermissions = app.service('/permissions/user');
+
+describe('userPermissions', async () => {
+	const ROLES = {
+		TEST: 'test',
+		OTHER: 'other',
+		EXTENDED: 'extended',
+		MULTIPLE: 'multiple',
+		XX: 'xx',
+		NOTHING: 'nothing spezial',
+  };
+  const testPermissions = [
+		'SINGING',
+		'DANCE_RAIN',
+		'WALK_LINES',
+		'RUN_FLOOR',
+  ];
+  
+  	const otherPermissions = [
+		'SITTING',
+		'SITTING_ON_CHAIR',
+		'SITTING_ON_DESK',
+	];
+  
+	before(async () => {
+		const testRole = await testObjects.createTestRole({
+			name: ROLES.TEST,
+			permissions: testPermissions,
+    });
+    
+    const testRoleWithDiffrentPermissons = await testObjects.createTestRole({
+			name: ROLES.TEST,
+			permissions: otherPermissions,
+		});
+
+		const otherRole = await testObjects.createTestRole({
+			name: ROLES.OTHER,
+			permissions: otherPermissions,
+		});
+
+    // TestSchool
+    const testSchool = await testObjects.createTestSchool({
+       permissions: [testPermissions],
+    })
+
+    const testSchoolWithoutRole = await testObjects.createTestSchool({
+    });
+
+    //TestUser
```

Comment:
Codacy found an issue: [Expected exception block, space or tab after '//' in comment.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5786177)